01_host/06_anv/collations.adoc

```diff
@@ -6,7 +6,7 @@ chain validators. The Polkodat network protocol is agnostic on what candidate
 productionis mechanism each parachain uses and does not specify or mandate any
 of such production methods (e.g. BABE-GRANDPA, Aura, etc). Furthermore, the
 relay chain validator host implementation itself does not directly interpret or
-process teh internal transactions of the candidate, but rather rely on the
+process the internal transactions of the candidate, but rather rely on the
```

Comment:
These misspellings should be fixed (and stay fixed) with https://github.com/w3f/polkadot-spec/pull/509 😉